### ISSUE TITLE
Allow non-healthchecks.io URLs

### ISSUE THREAD

**Issue by scott-8** [pfidr34/docker-rclone#26]
Instead of healthchecks.io, I've always used the URL fields to send a request to a different notification service. This was fairly easy, since all of the requests are sent through `wget $CHECK_URL -O /dev/null`, and I can add whatever I want to `CHECK_URL`. When the ability to send logs was added (#25), this functionality was broken.

For reference, my current `CHECK_URL` is `http://{SERVER}/message?token={TOKEN} --post-data=priority={INT}&message={MESSAGE}&title={TITLE}`.

One way to add back this functionality would be to create an environmental variable allowing the user to set their own post data, and if this variable is not set, just send the logs. Let me know if you think of a better way to implement this.

Also, do you think it'd be possible to add a separate `START_URL` parameter with it's own post data? And it it's blank, the `$CHECK_URL/start` could be used just like it is now.

To summarize, it'd be nice to have these variables:
- `START_URL`: url for start signal, if not set, use `CHECK_URL`
- `START_DATA`: post data for `START_URL`, if not set, send `START_URL/start`
- `CHECK_URL`: same as now
- `CHECK_DATA`: custom post data for `CHECK_URL`, if not set, send logs
- `FAIL_URL`: same as now
- `FAIL_DATA`: custom post data for `FAIL_URL`, if not set, send logs

All existing configurations should work the same with the above changes. Let me know what you think.

**Comment by dantebarba** [comment]
+1

**Comment by pfidr34** [comment]
@scott-8 @dantebarba 

Just wanted to give you an update that I have read this issue and I think it is a good idea. 

I am currently very busy at work and have not had time to look into this but may in the future. 

If you would like to submit a pull request for this feature I would also be happy to review that.
